components/templates/src/filters.rs

```diff
@@ -64,7 +73,17 @@ mod tests {
     fn markdown_filter() {
         let result = markdown(&to_value(&"# Hey").unwrap(), &HashMap::new());
         assert!(result.is_ok());
-        assert_eq!(result.unwrap(), to_value(&"<h1>Hey</h1>\n").unwrap());
+        // zola render adds an id to the h1 tag
+        assert_eq!(result.unwrap(), to_value(&"<h1 id=\"hey\">Hey</h1>\n").unwrap());
```

Comment:
Slight rendering change, the zola renderer inserts an id to the h1 tag. I considered this ok, and changed the test.